src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -599,38 +584,41 @@ export default class StripViewModel extends BaseModel {
     /**
      * Click handler for a single click on `StripViewModel`
      *
-     * This handler will prevent event bubbling
+     * This method will prevent event bubbling so a click
+     * doesn't cause the `stripView` to close
      *
      * @for AircraftStripView
-     * @method _onClickHandler
+     * @method _onClick
      * @param event {jquery event}
      * @private
      */
-    _onClickHandler = (event) => {
+    _onClick(event) {
         event.stopPropagation();
 
         this._eventBus.trigger(EVENT.STRIP_CLICK, this._callsign);
-    };
+    }
 
     /**
      * Handler for a double-click on an AircraftStripView
      *
      * Initiates a two-step event process, though undesired, is necessary.
      * We don't (and shouldn't) have access to the `AircraftController` or the
-     * `CanvasController` from within this class.
+     * `CanvasController` from within this class. A double click will trigger
+     * event listeners in both of those classes
      *
-     * This handler will prevent event bubbling
+     * This method will prevent event bubbling so a click
+     * doesn't cause the `stripView` to close
      *
      * @for AircraftStripView
-     * @method _onDoubleClickHandler
+     * @method _onDoubleClick
      * @param  event {jquery event}
      * @private
      */
-    _onDoubleClickHandler = (event) => {
+    _onDoubleClick(event) {
         event.stopPropagation();
 
         this._eventBus.trigger(EVENT.STRIP_DOUBLE_CLICK, this._callsign);
```

Comment:
I'm not sure how long it's been like this, but it would seem that the event bus callback for this event does _NOT_ mark the canvas as dirty, so we get this weird behavior where the center point and aircraft are moved, but the canvas stays static on the screen (and thus, is misaligned). Smells like we should be `CanvasController._markDeepRender()`ing somewhere on an `EVENT.STRIP_DOUBLE_CLICK` occurrence.

Reply:
This event is only triggering a handler in the `AircraftController` now.  Updated the docs about that.  Also updated `._onCenterPointInView()`, which is the eventual called method, to call `_markDeepRender()`.

Tested locally and didn't see any weirdness.